app/controllers/concerns/education_navigation_ab_testable.rb

```diff
@@ -4,10 +4,7 @@ def should_present_new_navigation_view?(content_item)
   end
 
   def present_taxonomy_sidebar?(content_item)
```

Comment:
Given that this method is now essentially an alias, it would be neater to remove the entire method and change any references to it to point to `should_present_new_navigation_view?` instead.